bfh/fields.py

```diff
@@ -63,7 +64,10 @@ def __get__(self, instance, cls=None):
         return instance.__dict__.get(self.field_name)
 
     def __set__(self, instance, value):
-        instance.__dict__[self.field_name] = value
+        if value is None and self.default:
```

Comment:
since you test falseyness of self.default `False` or `[]` specified as a default won't act as a user would expect